README.md

```diff
@@ -23,11 +23,14 @@ go get github.com/Clever/csvlint/cmd/csvlint
 
 ### Options
 
-  * delimiter: the field delimiter to default with
-    * default: comma
-    * valid options: comma, tab
-    * if you want anything else, you're probably doing CSVs wrong
-  * lazyquotes: allow a quote to appear in an unquoted field and a non-doubled quote to appear in a quoted field. _WARNING: your file may pass linting, but not parse in the way you would expect_
+_*WARNING:* to validate strictly against RFC 4180, use the defaults. However we understand that the real world might require a little more flexibility. Note that your file might not parse the way you expect if you do specify other options._
```

Comment:
I think this is a bit confusing. I might phrase it as:

```markdown
*NOTE*: The default settings test that a CSV conforms to RFC 4180. By changing the settings, you're no longer strictly adhering to RFC 4180.
```

And I might leave the warning about parsing associated with the `lazyquotes` option